data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	A slightly awkward half hour passes in which the uniqueness of your presence is more and more noted, before the thirty Saryds enter the arena and regain the crowd's attention. Holding what looks like fancy polo mallets, the members of each team are once more introduced to the crowd with a drone fly over, and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`
+			`	Once the game starts, you start to realize that it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defences. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`
+			`	Eventually, a sphere finds its way into a hole, which lights up red for the color of the scoring team. Much of the crowd jumps up and screams, though you can't quite tell if everyone is a particular fan of anyone on the red team or if they're cheering simply because a point was finally scored. The sphere is shot back out automatically from a tube close to the hole and is immediately back in play.`
+			`	As the game continues each team scores a few times, but the blue team pulls into the lead. When they score their fifth point, the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The helpful Kimek with the translator excitedly explains to you what is happening.`
+			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, starting with the yellow team, the lowest scoring, and progressing up to the blue team at the end. After this event, Ulmodi has risen to being placed third, and you notice that the Saryd who caught your eye in the forest earlier is in first place. After the individual scores are announced, the top twenty contestants celebrate by running laps around the arena.`
```

Comment:
```suggestion
			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, starting with the yellow team, the lowest scoring, and progressing up to the blue team at the end. After this event, Ulmodi, who was on the blue team, has risen to being placed third, and you notice that the Saryd who caught your eye for perfectly clearing the forest earlier is in first place. After the individual scores are announced, the top twenty contestants celebrate by running laps around the arena.`
```
It feels strange to not mention which team Ulmodi was on, and being on blue is the simplest/shortest explanation for why his place would rise. Added some detail to the first place Saryd since there is more than one unnamed Saryd mentioned (specifically the one who's disqualified). Not necessary, but I thought these two might help for clarification.